api/tests/src/services/opportunities_v1/test_opportunity_version.py

```diff
@@ -0,0 +1,55 @@
+from src.constants.lookup_constants import OpportunityCategory
+from src.db.models.opportunity_models import OpportunityVersion
+from src.services.opportunities_v1.opportunity_version import save_opportunity_version
+from tests.src.db.models.factories import (
+    AgencyFactory,
+    OpportunityAssistanceListingFactory,
+    OpportunityFactory,
+)
+
+
+def test_save_opportunity_version(db_session, enable_factory_create):
+    # Setup Opportunity
+    agency_top = AgencyFactory.create()
+    agency = AgencyFactory.create(top_level_agency=agency_top)
+
+    opp = OpportunityFactory.create(
+        opportunity_assistance_listings=[],
+        agency_code=agency.agency_code,
+        no_current_summary=True,
+        category=OpportunityCategory.MANDATORY,
+    )
+
+    opp_ass = OpportunityAssistanceListingFactory.create(opportunity=opp)
```

Comment:
If we want a slightly friendlier way to shorten it, I've seen "AL" for assistance listing
```suggestion
    opp_al = OpportunityAssistanceListingFactory.create(opportunity=opp)
```

Reply:
Done